src/test/kotlin/com/igorwojda/integer/stepsgenerator/challenge.kt

```diff
@@ -4,7 +4,17 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 fun generateSteps(n: Int): List<String> {
-    TODO("not implemented")
+    var str = ""
+    val steps = mutableListOf<String>()
+    (1..n).forEach { row ->
```

Comment:
You could use "map" instead of "forEach" and construct the list directly from the mapping construct.